api/TouchEvent.json

```diff
@@ -0,0 +1,522 @@
+{
+  "api": {
+    "TouchEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/TouchEvent",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": "22"
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true,
+            "flags": [
+              {
+                "type": "preference",
+                "name": "Standards Preview",
+                "value_to_set": "true"
+              }
+            ]
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": [
+            {
+              "version_added": "52"
+            },
+            {
+              "version_added": "18",
+              "version_removed": "24"
+            }
+          ],
+          "firefox_android": {
+            "version_added": "6"
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": false
+          },
+          "opera_android": {
+            "version_added": true
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": true
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "TouchEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/TouchEvent/TouchEvent",
+          "description": "<code>TouchEvent()</code> constructor",
+          "support": {
+            "webview_android": {
+              "version_added": "48",
+              "notes": "Chrome only supports the following <code>touchEventInit</code> properties: <code>touches</code>, <code>targetTouches</code>, <code>changedTouches</code>."
+            },
+            "chrome": {
+              "version_added": "48",
+              "notes": "Chrome only supports the following <code>touchEventInit</code> properties: <code>touches</code>, <code>targetTouches</code>, <code>changedTouches</code>."
+            },
+            "chrome_android": {
+              "version_added": "48",
+              "notes": "Chrome only supports the following <code>touchEventInit</code> properties: <code>touches</code>, <code>targetTouches</code>, <code>changedTouches</code>."
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": true
```

Comment:
It's difficult to know which table to look at (https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent/TouchEvent or https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent) but I usually pick the one on the separate page and stick to that data. That said, I understand why you referred to the base page when the data was absent on the other.

Reply:
My interpretation of this is that https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent refers to the API as a whole, but https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent/TouchEvent refers specifically to the TouchEvent constructor.

That is, it's possible to listen for touch events (thus using the TouchEvent API) without being able to synthesize them in a script (by calling the `TouchEvent()` constructor).

Is this not correct?